### PR TITLE
Development

### PR DESCRIPTION
**Pull request by JamesOkunlade**
Library Project

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
+/// Add New Book Modal scripts
+
+var modal = document.querySelector(".modal");
```
Comment (dipto0321) [inline js/script.js]: Is that possible here to use `const` or `let` ?

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+var trigger = document.querySelector(".trigger");
```
Comment (dipto0321) [inline js/script.js]: Same like ☝️

File: js/script.js [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+var closeButton = document.querySelector(".close-button");
```
Comment (dipto0321) [inline js/script.js]: same like ☝️

File: js/script.js [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+        Store.toggleRead(e.target.previousElementSibling.textContent);
+    }
+  });
+
+
+/// Deleting books
+document.querySelector('#shelf').addEventListener('click', function(e) {
+  if (e.target.classList.contains('delete')) {
+    e.target.parentElement.parentElement.parentElement.remove();
+    console.log(e.target.parentElement.previousElementSibling.textContent);
```
Comment (dipto0321) [inline js/script.js]: remove console log

**Comment by JamesOkunlade** [comment]
Hello @dipto0321 
Thanks for the prompt review. The error you have is not evident here. Here is a picture of what it looks like after adding a book to the library..

image

Please kindly check again or try another browser. We use localStorage in our project and some obsolete browsers don't work well with localStorage. 

Meanwhile, I have corrected the other reviews and submitted for another review. 

Thanks

**Comment by dipto0321** [comment]
Hi @JamesOkunlade  and @AdnanAfsari ,

According to our slack discussion, I already provide you the feedback. So after fixing "Add button issue"  issue please send another request to re-review this project.

Thanks 🙏

**Comment by JamesOkunlade** [comment]
@dipto0321 ,

We have fixed the error. We were able to debug the error only after installing http-server to the project and found out the book-form was doing `POST`  twice.

I hope we get a :+1: now. :sunglasses:

**Comment by dipto0321** [comment]
HI @JamesOkunlade and @AdnanAfsari ,

So far so good. Finally, you did it 😃 . This is from me 👍 😎 

Thanks 🙏 😌

**Pull request merged**